sympy/combinatorics/perm_groups.py

```diff
@@ -2567,6 +2567,78 @@ def minimal_block(self, points):
         new_reps = {}
         return [new_reps.setdefault(r, i) for i, r in enumerate(parents)]
 
+    def conjugacy_class(self, x):
+        r"""Return the conjugacy class of an element in the group.
+        The conjugacy class of an element ``g`` in a group ``G`` is the set of
+        elements ``x`` in ``G`` that are conjugate with ``g``, i.e. for which
+            ``g = xax^{-1}``
+        for some ``a`` in ``G``.
+        Note that conjugacy is an equivalence relation, and therefore that
+        conjugacy classes are partitions of ``G``. For a list of all the
+        conjugacy classes of the group, use the conjugacy_classes() method.
+        In a permutation group, each conjugacy class corresponds to a particular
+        `cycle structure': for example, in ``S_3``, the conjugacy classes are:
+            * the identity class, ``{()}``
+            * all transpositions, ``{(1 2), (1 3), (2 3)}``
+            * all 3-cycles, ``{(1 2 3), (1 3 2)}``
+        Examples
+        ========
```

Comment:
https://github.com/sympy/sympy/pull/18350#issuecomment-575004951
I mean in the original one, it was spaced like 
```
 * all 3-cycles, ``{(1 2 3), (1 3 2)}``

Examples
========

>>> from sympy.combinatorics.permutations import Permutation
```
And now, you have made them less readable.